doc/sphinx-guides/source/installation/oidc.rst

```diff
@@ -80,9 +116,64 @@ The Dataverse installation will automatically try to load the provider and retri
 You should see the new provider under "Other options" on the Log In page, as described in the :doc:`/user/account`
 section of the User Guide.
 
-By default, the Log In page will show the "builtin" provider, but you can adjust this via the ``:DefaultAuthProvider``
-configuration option. For details, see :doc:`config`.
-
-.. hint::
-   In contrast to our :doc:`oauth2`, you can use multiple providers by creating distinct configurations enabled by
-   the same technology and without modifying the Dataverse Software code base (standards for the win!).
+.. _oidc-mpconfig:
+
+Provision via MPCONFIG
+^^^^^^^^^^^^^^^^^^^^^^
+
+In case you only require a single OIDC provider, you can also provision an OIDC based provider using MicroProfile Config
```

Comment:
Is it premature to consider supporting multiple OIDC providers, perhaps like how we support multiple file stores?

Reply:
I agree with Oliver's comment, especially if the authorization (groups, ACLs) is also managed by an "OIDC server".

Reply:
@pdurbin can we resolve this?